nautobot_floor_plan/svg.py

```diff
@@ -439,7 +439,7 @@ def _draw_rack_tile(self, drawing, tile):
                 tile.rack.name,
                 insert=(
                     origin[0] + (tile.x_size * self.GRID_SIZE_X) / 2,
-                    origin[1] + (tile.y_size * self.GRID_SIZE_Y) / 2 - self.TEXT_LINE_HEIGHT,
+                    origin[1] + (tile.y_size * self.GRID_SIZE_Y) / 2.5 - self.TEXT_LINE_HEIGHT,
```

Comment:
`(tile.y_size * self.GRID_SIZE_Y) / 2` is the vertical center of the tile, right? Not clear to me what dividing by 2.5 instead does to the positioning. Should the change here be something like `- 2 * self.TEXT_LINE_HEIGHT` or `-1.5 * self.TEXT_LINE_HEIGHT` instead of changing the divisor?